updater/Cargo.toml

```diff
@@ -28,3 +28,9 @@ rand = "0.4"
 ethcore = { path = "../ethcore", features = ["test-helpers"] }
 tempdir = "0.3"
 matches = "0.1"
+
+[features]
+# use harcoded version 1.3.7 of parity to force an update
```

Comment:
What is "magic" about v1.3.7? Might be worth mentioning in the comment?

Reply:
Nothing, just old enough to be regarded as a critical update!

Reply:
I'd add a parenthesis like "(a version old enough to require a critical update, nothing special about it per se)" or something along those line. But up to you!